core/src/test/java/org/apache/calcite/rex/RexProgramTest.java

```diff
@@ -1774,6 +1770,17 @@ private void checkExponentialCnf(int n) {
     checkSimplify2(e, "SEARCH(?0.int0, Sarg[10])", "=(?0.int0, 10)");
   }
 
+  @Test void testSimplifyInAnd() {
```

Comment:
Maybe we need another test case: 

deptno in (20, 10) and deptno = 30 
==> false?